DTS-STN/canadian-dental-care-plan
# Removing federal/provincial benefits lookup service and corresponding mocks/configuration

### Description
Follow-up PR for https://github.com/DTS-STN/canadian-dental-care-plan/pull/903, there is now no need for a lookup service function to get the federal/provincial benefits options.

Cache time configuration and unused Lookup API/DB mocks have also been removed.

### Checklist

### Test Instructions
Run the application and start a CDCP application. Continue the application until you reach `/federal-provincial-territorial-benefits`. There should be no issues with the "Yes"/"No" options when submitting this page and the result of your selection should show on the `/review-information` page.
